tests/auth_tests/test_forms.py

```diff
@@ -1371,3 +1402,20 @@ def test_html_autocomplete_attributes(self):
                 self.assertEqual(
                     form.fields[field_name].widget.attrs["autocomplete"], autocomplete
                 )
+
+    def test_unusable_password(self):
+        """Tests if deselecting usable password sets an unusable password"""
+        user = User.objects.get(username="testclient")
+        form = AdminPasswordChangeForm(
+            user,
+            {
+                "password1": "",
+                "password2": "test",
+            },
+        )
+
+        valid = form.is_valid()
+        form.save(commit=True)
+
+        self.assertIs(valid, True)  # Valid despite password empty/mismatch.
+        self.assertIs(user.has_usable_password(), False)
```

Comment:
Can we also add assertions for `password`.

Reply:
The only assertion I can think of is that `user.password` starts with `UNUSABLE_PASSWORD_PREFIX`. I feel this is somewhat redundant to `user.has_usable_password()` being `False´, but certainly, I can add this. Or were you thinking of something else?

Reply:
Right, probably not needed.